lib/routes/account.js

```diff
@@ -487,13 +489,20 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            getGeoData(ip)
+              .then(
+                function (loc) {
+                  mailer.sendNewDeviceLoginNotification(
+                    emailRecord.email,
+                    userAgent.call({
+                      acceptLanguage: request.app.acceptLanguage,
+                      ip: ip,
+                      location: loc.location,
```

Comment:
re: https://github.com/mozilla/fxa-auth-server/pull/1334/files#r72123080 

seems like `geoData` here might better here because it also includes the time zone